sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java

```diff
@@ -115,10 +115,18 @@ public static boolean hasResourceId(Resource resource) {
     }
 
     public static Optional<String> getSourceRoot(Resource resource) {
```

Comment:
Should VolumeEvaluationStage be calling this? Also, can't hurt to check its current callers (there could very well not be any)

Reply:
WIP artifact.   Should be called appropriately now.